lib/matplotlib/tests/test_artist.py

```diff
@@ -140,30 +148,50 @@ def test_cull_markers():
 
 @image_comparison(['hatching'], remove_text=True, style='default')
 def test_hatching():
+    plt.rcParams['hatch.linewidth'] = 9
     fig, ax = plt.subplots(1, 1)
 
     # Default hatch color.
-    rect1 = mpatches.Rectangle((0, 0), 3, 4, hatch='/')
+    rect1 = mpatches.Rectangle((0, 0), 3, 4, hatch='/', lw=4)
     ax.add_patch(rect1)
 
     rect2 = mcollections.RegularPolyCollection(4, sizes=[16000],
-                                               offsets=[(1.5, 6.5)],
+                                               offsets=[(1.5, 5.5)],
                                                transOffset=ax.transData,
-                                               hatch='/')
+                                               hatch='/', lw=4)
     ax.add_collection(rect2)
 
     # Ensure edge color is not applied to hatching.
```

Comment:
I don't get why this comment is here: as it was and as it is now the edge color _does_ get applied to the hatching.

Reply:
That comment should be removed.  Doing a bit of git-blame digging and not removing that was an error on my part in https://github.com/matplotlib/matplotlib/pull/7976 (which restored hatches folliowing the edges see for bug report: https://github.com/matplotlib/matplotlib/issues/7901).  The hatches did not follow edges for 1 minor release via https://github.com/matplotlib/matplotlib/pull/7415 which is where the comment came in.